### PR TITLE
Modified some content to for flow and acuracy

### PR DESCRIPTION
**Pull request by ijiraq**
My writing is SSSOOOOO much better than your's.

### CODE REVIEW

File: Survey_Simulator/README.md [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
-
-TNO surveys have a number of strong observational biases. In order to meaningfully interpret the observations, these biases must be taken into consideration so that conclusions can be drawn about the intrinsic TNO populations being observed. Each of the above surveys has been carefully characterised and has measured their intrinsic biases. These biases are recorded in the \*.eff files stored in the directory "ALL". The Survey Simulator takes a straightforward approach to handling the observational biases. It is not possible to take the observed population and de-bias them to know what the intrinsic populations is. One simply can't know what they didn't see. Instead, the Survey Simulator takes a model for the intrinsic distribution of objects in the Solar System and applies the survey's known biases. A model is forward biased through the Survey Simulator to produce a set of "Simulated Detections" that can then be compared to the real detected sample. For details on the inner workings of the Survey Simulator, see the following paper:
+This document provides description of the SurveySimulator package.
```
Comment (cshankm) [inline Survey_Simulator/README.md:1]: provides a description?

File: Survey_Simulator/README.md [hunk 2]
```diff
@@ -1,20 +1,36 @@
...
+The SurveySimulator software package simulates the detection of a model distribution of Trans-Neptunian Objects (TNOs) subject to the detection characteristics of a given survey. 
+This package proivides the SureveySimulator software as well as the characterization files for the three TNO surveys: [CFEPS](http://adsabs.harvard.edu/abs/2011AJ....142..131P), [Alexandersen et al. 2015](http://arxiv.org/abs/1411.7953) and [OSSOS](http://www.ossos-survey.org/). 
+This version of the package is offered as a companion to Shankman et al. 2015, (an analyses the H magnitude distribution Scattering TNOs)
```
Comment (cshankm) [inline Survey_Simulator/README.md:4]: "an analyses" --> "an analysis of"
distribution --> add "of"

File: Survey_Simulator/README.md [hunk 3]
```diff
@@ -1,20 +1,36 @@
...
+
+TNO surveys have a number of strong observational biases. 
+To draw meaningfully conclusions regarding the intrinsic distribution of TNOs these biases must be taken into consideration.
```
Comment (cshankm) [inline Survey_Simulator/README.md:7]: comma after "TNOs"

File: Survey_Simulator/README.md (hunk 1 above)
Comment (ijiraq) [inline Survey_Simulator/README.md:1]: yes.  I think you can make all these proposed changes.  I might have time to look at this on Monday, or you can just merge my changes and then correct.

**Pull request merged**